src/ml/neural_net/tf_compute_context.hpp

```diff
@@ -47,6 +67,22 @@ class tf_compute_context: public compute_context {
   
 };
 
+class tf_model_backend : public model_backend {
```

Comment:
I don't think this even needs to be in the header file. If we move this to the top of the `.cpp` file, then all the pybind headers can go there too

Reply:
Doing it in the other PR.